lib/fizzy/execute.cpp

```diff
@@ -271,6 +271,27 @@ Instance instantiate(const Module& module, std::vector<ImportedFunction> importe
     std::vector<TypeIdx> imported_function_types =
         match_imports(module, imported_functions, imported_globals);
 
+    // Set up tables
+    std::vector<FuncIdx> table;
+    if (!module.tablesec.empty())
+        table.resize(module.tablesec[0].limits.min);
+
+    assert(module.elementsec.empty() || !module.tablesec.empty());
+    for (const auto& element : module.elementsec)
+    {
+        uint64_t offset;
+        if (element.offset.kind == ConstantExpression::Kind::Constant)
+            offset = element.offset.value.constant;
+        else
+            throw std::runtime_error(
+                "element initialization by imported global is not supported yet");
+
+        // Overwrite table[offset..] with element.init
+        assert((offset + element.init.size()) <= table.size());
+        std::copy(element.init.begin(), element.init.end(), &table[offset]);
+    }
+
+    // TODO: add imported globals first
```

Comment:
Remove this, it's already done, imported globals are in `imported_globals`

Reply:
And now I have to support that :(

I hoped to merge this before the imported globals was merged :)

Reply:
(I do this for the data section in https://github.com/wasmx/fizzy/pull/100, probably will add elemts there, too, if we merge this one first)

Reply:
Removed.